spec/support/medium_spec.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.shared_examples "a medium" do
+
+  describe "model validations" do
+    it "must have a name" do
+      medium = described_class.new
+      expect(medium).to_not be_valid
+      expect(medium.errors.keys).to include :name
```

Comment:
Excellent use of `errors.keys` to check that the reason(s) our medium is not valid actually includes the reason we expect!